blog/electron-21-0.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Electron 21.0.0
+date: 2022-09-27T00:00:00.000Z
+authors:
+    - name: ckerr
+      url: 'https://github.com/ckerr'
+      image_url: 'https://github.com/ckerr.png?size=96'
+    - name: vertedinde
+      url: 'https://github.com/vertedinde'
+      image_url: 'https://github.com/vertedinde.png?size=96'
+    - name: georgexu99
+      url: 'https://github.com/georgexu99'
+      image_url: 'https://github.com/georgexu99.png?size=96'
+slug: electron-21-0
+
+---
+
+Electron 20.0.0 has been released! It includes upgrades to Chromium `106`, V8 `10.4`, and Node.js `16.16.0`. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 21.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://www.electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
+
+## Notable Changes
+
+### New Features
+
+TODO: Add features here
+
+### Stack Changes
+
+* Chromium `106`
+    * [New in Chrome 106](https://developer.chrome.com/blog/new-in-chrome-106/)
+    * [New in Chrome 105](https://developer.chrome.com/blog/new-in-chrome-105/)
+    * [New in DevTools](https://developer.chrome.com/blog/new-in-devtools-106/)
```

Comment:
Do we also want to link https://developer.chrome.com/blog/new-in-devtools-105/ ?

Reply:
In previous posts, we just did the latest DevTools release, but I honestly don't see a reason to not do both - I'll pop that in.